pkg/model/api.go

```diff
@@ -51,6 +53,58 @@ func ImportModel(ctx context.Context, filePath, projectId, message, modelType, b
 		return err
 	}
 
-	log.Println("Starting import model job. JobId: ", importModelData.GetImportModelJobId())
+	importModelJobId := importModelData.GetImportModelJobId()
+	if waitForResults {
+		okStatus, modelId, err := waitForImportModelJob(ctx, importModelJobId)
+		if err != nil {
+			return err
+		}
+		if okStatus {
+			log.Println("Successfully imported model. ModelId: ", modelId)
+		} else {
+			log.Println("Failed to import model")
```

Comment:
Consider printing the failure message - it is essential to understand what the problem is with the model

Reply:
Added

Reply:
I mean print the error if it is possible